PocketKit/Sources/PocketKit/Settings/AppIcon/SelectIconView.swift

```diff
@@ -0,0 +1,65 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Localization
+import SwiftUI
+import Textile
+
+struct SelectIconView: View {
+    @StateObject var viewModel = SelectIconViewModel()
+
+    var body: some View {
+        Form {
+            Group {
+                Section {
+                    List(PocketAppIcon.allCases) { appIcon in
+                        HStack(spacing: 16) {
+                            Image(appIcon.previewName)
+                                .resizable()
+                                .aspectRatio(contentMode: .fit)
+                                .frame(width: 60, height: 60)
+                                .cornerRadius(12)
+                            Text(appIcon.description)
+                            Spacer()
+                            CheckboxView(isSelected: viewModel.selectedAppIcon == appIcon)
```

Comment:
Let's add analytics for which app icon they select

We can do an engagement event and for the detail, put the icon name.

Reply:
done